lib/modules/ssh/strong_password/__init__.py

```diff
@@ -1,7 +1,35 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
+import time
 from core.compatible import generate_token
+from database.connector import insert_honeypot_events_from_module
+from os import path
+
+
+class ModuleProcessor:
+    """
+    this is the processor to run after docker machine is up to grab the log files or do other needed process...
+    """
+    def __init__(self):
+        self.kill_flag = False
+        self.pointer=''
+
+    def processor(self):
+        """
+        processor function will be called as a new thread and will be die when kill_flag is True
+        :return:
+        """
+        open('tmp/logins.txt', 'w').close() #empty the previous logs if any
+        while not self.kill_flag:
+            if(path.exists('tmp/logins.txt')):
```

Comment:
you should check while true(if filesize > 0, then move file, newfile, read newfile, sumbit the logs, remove the new file, sleep).

Reply:
@Ali-Razmjoo : this strategy causes resource problem until i move file and read there are new requests in that file which are not read and when i remove the file they would also be lost.
we need to discuss another strategy for this.